go/vt/vtgate/planbuilder/show.go

```diff
@@ -603,8 +603,12 @@ func buildWarnings() (engine.Primitive, error) {
 		rows := make([][]sqltypes.Value, 0, len(warns))
 
 		for _, warn := range warns {
+			txt := "Warning"
+			if warn.Code == 1003 {
+				txt = "Note"
+			}
 			rows = append(rows, []sqltypes.Value{
-				sqltypes.NewVarChar("Warning"),
+				sqltypes.NewVarChar(txt),
```

Comment:
I have two things here.
1. I think we should keep the error code in go/mysql/constants.go
2. Do we ever log 1003? 
I know this will work when we start getting warnings from vttablet, just wanted to understand we are doing any logging with this code